applications/openshift/worker/file_owner_worker_service/rule.yml

```diff
@@ -19,12 +29,31 @@ severity: medium
 references:
     cis: 4.1.2
 
-ocil_clause: '{{{ ocil_clause_file_owner(file="/etc/systemd/system/atomic-openshift-node.service", owner="root") }}}'
+ocil_clause: |-
+{{%- if product == "ocp4" %}}
+  '{{{ ocil_clause_file_owner(file="/etc/systemd/system/kubelet.service", owner="root") }}}'
+{{% else %}}
+  '{{{ ocil_clause_file_owner(file="/etc/systemd/system/atomic-openshift-node.service", owner="root") }}}'
+{{%- endif %}}
 
-ocil: '{{{ ocil_file_owner(file="/etc/systemd/system/atomic-openshift-node.service", owner="root") }}}'
+ocil:
+{{%- if product == "ocp4" %}}
+  '{{{ ocil_file_owner(file="/etc/systemd/system/kubelet.service", owner="root") }}}'
+{{% else %}}
+  '{{{ ocil_file_owner(file="/etc/systemd/system/atomic-openshift-node.service", owner="root") }}}'
+{{%- endif %}}
 
+
+{{%- if product == "ocp4" %}}
+template:
+    name: file_owner
+    vars:
+        filepath: /etc/systemd/system/kubelet.service
+        fileuid: '0'
+{{% else %}}
 template:
     name: file_owner
     vars:
         filepath: /etc/systemd/system/atomic-openshift-node.service
         fileuid: '0'
+{{%- endif %}}
```

Comment:
@JAORMX @jhrozek Do I understand it correctly that this rule is expected to be present and behave differently for `ocp4` and `ocp4-node`?

Hey, I thought the rule sets  for `ocp4` and `ocp4-node` mutually exclusive. That a rule present in one wouldn't be in the other...